lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -652,35 +642,32 @@ def model_zones_with_occ_and_fuel_type(model, custom)
       zn_hash['bldg_type'] = thermal_zone_building_type(zone)
 
       # Fuel type
-      if /prm/i !~ template
-        # for 2013 and prior, baseline fuel = proposed fuel
-        zn_hash['fuel'] = thermal_zone_fossil_or_electric_type(zone, custom)
-      else
-        # for 2016 and later, use fuel to identify zones with district energy
-        zone_fuels = ''
-        htg_fuels = zone.heating_fuels
-        if htg_fuels.include?('DistrictHeating')
-          zone_fuels = 'DistrictHeating'
-        end
-        clg_fuels = zone.cooling_fuels
-        if clg_fuels.include?('DistrictCooling')
-          zone_fuels += 'DistrictCooling'
-        end
-        zn_hash['fuel'] = zone_fuels
-      end
+      # for 2013 and prior, baseline fuel = proposed fuel
+      # for 2016 and later, use fuel to identify zones with district energy
+      zn_hash['fuel'] = get_zone_fuels_for_occ_and_fuel_type(zone)
+
       zones << zn_hash
     end
 
     return zones
   end
 
+  # for 2013 and prior, baseline fuel = proposed fuel
+  # @param themal_zone
+  # @return [string] with applicable DistrictHeating and/or DistrictCooling 
+  def get_zone_fuels_for_occ_and_fuel_type(zone)
+    zone_fuels = thermal_zone_fossil_or_electric_type(zone, 'custom')
```

Comment:
Why are we use `'custom'` instead of `custom`, it looks like the original call used `custom`.

Reply:
The custom variable is not actually used in the thermal_zone_fossil_or_electric_type() method. So rather than dragging it through, I just put in a string. I'll change it to an empty string to make it more clear what is happening.

Reply:
I've moved this method to Standards.ThermalZone.rb to be consistent with the 90_1_prm version.